classes/Uploader.php

```diff
@@ -203,10 +203,10 @@ protected function checkUploadError($error_code)
 		switch ($error_code)
 		{
 			case 1:
-				$error = Tools::displayError(sprintf('The uploaded file exceeds %s', ini_get('upload_max_filesize')));
+				$error = sprintf(Tools::displayError('The uploaded file exceeds %s', ini_get('upload_max_filesize')));
```

Comment:
Hi,

Thank you for your PR, missing parenthesis after string in Tools::displayError !

Best regards